tests/providers/amazon/aws/transfers/test_mongo_to_s3.py

```diff
@@ -60,15 +60,20 @@ def setUp(self):
     def test_init(self):
         assert self.mock_operator.task_id == TASK_ID
         assert self.mock_operator.mongo_conn_id == MONGO_CONN_ID
-        assert self.mock_operator.s3_conn_id == S3_CONN_ID
+        assert self.mock_operator.aws_conn_id == AWS_CONN_ID
         assert self.mock_operator.mongo_collection == MONGO_COLLECTION
         assert self.mock_operator.mongo_query == MONGO_QUERY
         assert self.mock_operator.s3_bucket == S3_BUCKET
         assert self.mock_operator.s3_key == S3_KEY
         assert self.mock_operator.compression == COMPRESSION
 
     def test_template_field_overrides(self):
-        assert self.mock_operator.template_fields == ['s3_key', 'mongo_query', 'mongo_collection']
+        assert self.mock_operator.template_fields == (
+            's3_bucket',
+            's3_key',
+            'mongo_query',
+            'mongo_collection'
```

Comment:
```suggestion
            'mongo_collection',
```
This will make black happy
https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html